content/hardware/08.edu/solution-and-kits/alvik/tutorials/user-manual/user-manual.md

```diff
@@ -112,133 +135,1280 @@ Alvik has five different sensors, all connected to the STM32 and accessible thro
-2. The [latest libraries](https://github.com/arduino/arduino-alvik-mpy/releases) have to be placed in the Nano ESP32.
-3. The [latest firmware](https://github.com/arduino-libraries/Arduino_AlvikCarrier/releases) has to be uploaded to the STM32 microcontroller.
+Alvik is equipped with three ready-to-go examples. To choose one of the examples, just turn your Alvik ON, move the switch located at the bottom right corner of the robot to the right, wait until the LEDs turn blue and use the Up and Down buttons to pick one color, then hit the "tick" confirmation button. 
 
-Alvik comes with a preinstalled version of libraries and firmware, but in case you want to upgrade it or if something happens and you mess things up, here is the guide to reinstall both libraries and firmware.
+### Basic Touch Programming
 
-### MicroPython Firmware on the Nano ESP32
+Alvik’s touch buttons can be programmed to control various actions such as movement, changing modes, and interacting with the environment. This allows for creating intuitive control schemes for different applications. By default, the touch buttons are pre-programmed to control the Alvik's basic movements, such as moving forward, backward, turning left, and turning right. This setup provides an immediate interactive experience, enabling you to get started with minimal setup.
 
-Download and install the [Arduino Lab for MicroPython](https://labs.arduino.cc/en/labs/micropython), if you are able to connect the Arduino Nano ESP32 it means that you're board is ready. You have to see the **CONNECTED** yellow label at the bottom.
+### Following Objects
 
-![Connection Succeeded](assets/connection_succeeded.png)
+Out of the box, Alvik is equipped with distance sensors that allow it to follow objects by maintaining a certain distance from them. This feature is pre-configured to help the Alvik follow a target object, making it useful for applications like automated guided vehicles and interactive robotics. The default settings ensure that Alvik can smoothly track and follow objects placed in front of it without any additional programming required.
 
-If something goes wrong it means you need to upload the MicroPython firmware on the Nano ESP32.
+### Line Follower
 
-Follow [this guide](https://docs.arduino.cc/micropython/basics/board-installation/).
+Alvik comes pre-programmed with a line following algorithm that utilizes its built-in line follower sensors. These sensors enable the Alvik to navigate along predefined paths marked by lines on the ground. This feature is particularly beneficial in educational settings and for demonstrating basic robotic navigation. The default configuration allows Alvik to follow a black line on a white surface right out of the box, providing an engaging and interactive learning experience for you.
+
+## Alvik's Firmware
+
+To get started to play with Alvik you will need the following hardware and software:
 
-### How to Upload Libraries
+### Hardware Requirements

```

Comment:
this section is repeated 3 times in the manual, I think having it on the 1st section, in the unboxing, is enough

Reply:
Removed multiple listing of the components this is indeed a good point!